### DISCUSSION TITLE
Error Compiling Account Abstraction on ZkSync

### DISCUSSION THREAD

**Discussion by SpringxDSay** [Cyfrin/foundry-full-course-cu#2558] [Q&A] [answered]
Working on zkSync for the account abstraction project, I installed the zksync version of foundry and ran the "forge build --zksync" but got:
```
god@GODs-MacBook-Air foundry-account-abstraction % forge build --zksync
[⠃] Using zksolc-1.5.3
[⠒] Compiling (zksync)
[⠆] Compiling 84 files with zksolc and solc 0.8.25
Error: 
Broken pipe (os error 32)
```
What could be wrong?

**Comment by EngrPips** [comment, marked as answer]
Hello @SpringxDSay, Here https://github.com/matter-labs/foundry-zksync/issues/453 is a discussion on the official `foundry-zksync` repo that touches on this issue. Maybe check the discussion and every other discussion that touches on this issue to see if you will find a solution. This is also an Ethereum-stack-exchange post https://stackoverflow.com/questions/11866792/how-to-prevent-errno-32-broken-pipe that might help.

**Reply by SpringxDSay** [reply]
I'll check it out. Ty

**Reply by EngrPips** [reply]
You are welcome, friend.

**Reply by Iammrgemini** [reply]
> I'll check it out. Ty

Where you able to fix it?
